### PR TITLE
Fix casing of JSON in Jellyfin API

### LINKED ISSUE #2812: Startup Wizard doesn't save UI language correctly
**Describe the bug**
Selecting display language in the startup wizard does not correctly set the language. After selecting French in the setup, upon navigating to the dashboard, the display language is set to French.

**System (please complete the following information):**
 - OS: Windows 10
 - Virtualization: None
 - Clients: Browser
 - Browser: Firefox 74
 - Jellyfin Version: 10.5.2
 - Playback: N/A
 - Installed Plugins: None
 - Reverse Proxy: None
 - Base URL: None
 - Networking: Host
 - Storage: Local

**To Reproduce**
1. Start with a fresh install of Jellyfin
2. Click on any language in the setup wizard under Display Language
3. Finish setup
4. Go to the dashboard, and notice that translation is not occurring, and under settings->general, the language you selected is not listed as the display language.

**Expected behavior**
Homepage & dashboard will be correctly translated, and the previously selected language will appear under "Preferred display language"

**Additional context**
I've tracked down the source of the bug, the backend is looking for `uiCulture`, but the frontend is sending `UICulture`, will open PR on frontend to fix.

### PR DESCRIPTION
**Pull request by ZadenRB**
**Changes**
Fixes responses from the Jellyfin API being automatically camel cased. Currently only affects the startup wizard endpoints.
**Issues**
Fixes #2812 
Replaces jellyfin/jellyfin-web#1050

### CODE REVIEW

File: Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -71,6 +71,10 @@ public static IMvcBuilder AddJellyfinApi(this IServiceCollection serviceCollecti
                 // Clear app parts to avoid other assemblies being picked up
                 .ConfigureApplicationPartManager(a => a.ApplicationParts.Clear())
                 .AddApplicationPart(typeof(StartupController).Assembly)
+                .AddJsonOptions(options => 
+                {
+                    options.JsonSerializerOptions.PropertyNamingPolicy = null;
```
Comment (JustAMan) [inline Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs:76]: what does this do?

Comment (ZadenRB) [reply Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs:76]: This line is what actually fixes the issue. The changes to the property names in the other file were just for clarity regarding what the endpoint expects. The default JSON options use a the camel case property naming policy, so when we return `StartupCollectionDto`, it takes every property of that object and camel cases it. Setting the naming policy to null leaves the property names unchanged.

Comment (JustAMan) [reply Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs:76]: Hmm, this would make an excellent comment right there.

Comment (ZadenRB) [reply Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs:77]: Good point. Added!

**Pull request merged**